nserror-kt/src/nativeInterop/cinterop/NSErrorKtRuntime.def

```diff
@@ -1,12 +1,54 @@
 ---
+
+#include "stdint.h"
+#include "stddef.h"
+
+// Based on Kotlin Native runtime
+
+typedef struct ObjHeader ObjHeader;
+
+typedef struct FrameOverlay {
```

Comment:
This `FrameOverlay ` struct must exactly match the one from the Kotlin Native runtime, right?
In older versions it also had a `void* arena;` which was removed in 1.9.20.
P.S. I will update the library to Kotlin 1.9.22, so no need to add it.

Reply:
> This FrameOverlay  struct must exactly match the one from the Kotlin Native runtime, right

More or less. Ideally, it should be - just to be sure everything works correctly. However, based on what I have seen in the runtime, the presence or absence of this specific field doesn't seem to matter for this specific use case.

> I will update the library to Kotlin 1.9.22, so no need to add it.
Great! However, be aware that what matters in this case is the version of the runtime, not the language. And the runtime version is dictated by the final binary.